src/main/java/com/abcdedu_backend/infra/file/S3FileHandler.java

```diff
@@ -32,28 +31,55 @@ public class S3FileHandler implements FileHandler {
     private String bucketName;
 
     @Override
-    public String upload(MultipartFile file, FileDirectory directory, String fileName){
-        try (InputStream inputStream = file.getInputStream()){
+    public String upload(MultipartFile file, FileDirectory directory, String fileName) {
+        try (InputStream inputStream = file.getInputStream()) {
             String extension = getExtension(file);
             String key = createUploadKey(directory, fileName, extension);
             log.info("S3 upload 성공 key: {}", key);
-            amazonS3.putObject(new PutObjectRequest(bucketName, key, inputStream,null));
+            amazonS3.putObject(new PutObjectRequest(bucketName, key, inputStream, null));
             return key;
         } catch (Exception e) {
             log.error("S3 upload 실패 : {}", e.getLocalizedMessage());
             throw new ApplicationException(ErrorCode.S3_UPLOAD_ERROR);
         }
     }
 
+//    @Override
+//    public String getPresignedUrl(String objectKey) {
+//        Date expiration = getDateOneHourLater();
+//
+//        GeneratePresignedUrlRequest generatePresignedUrlRequest = new GeneratePresignedUrlRequest(bucketName, objectKey)
+//                .withMethod(HttpMethod.GET)
+//                .withExpiration(expiration);
+//
+//        return amazonS3.generatePresignedUrl(generatePresignedUrlRequest).toString();
+//    }
+
     @Override
     public String getPresignedUrl(String objectKey) {
+        GeneratePresignedUrlRequest generatePresignedUrlRequest = generatePresignUrlRequestForExtension(objectKey);
+        log.info("generatePresignedUrlRequest 생성 완료");
+        return amazonS3.generatePresignedUrl(generatePresignedUrlRequest).toString();
+    }
+
+    private GeneratePresignedUrlRequest generatePresignUrlRequestForExtension(String objectKey) {
+
```

Comment:
```suggestion
    private GeneratePresignedUrlRequest getGeneratePresignUrlRequestForExtension(String objectKey) {

```
p4: 별거는 아니지만 안에서 getExtension처럼 get으로 명확하게 표현하면 좋을 것같아요.